<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CorrelationWidget</class>
 <widget class="QWidget" name="CorrelationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>641</width>
     <height>681</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="formTitleLabel">
      <property name="text">
       <string>Form Title</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="correlationTab">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Correlations</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>10</y>
          <width>241</width>
          <height>471</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item alignment="Qt::AlignHCenter">
            <widget class="QLabel" name="correlationQuestionLabel2">
             <property name="text">
              <string>Question 2</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="correlationQuestion2">
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <property name="isWrapping" stdset="0">
              <bool>false</bool>
             </property>
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item alignment="Qt::AlignHCenter">
            <widget class="QLabel" name="correlationAnswerLabel2">
             <property name="text">
              <string>Answer Choice 2</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="correlationAnswer2">
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::MultiSelection</enum>
             </property>
             <property name="isWrapping" stdset="0">
              <bool>false</bool>
             </property>
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>241</width>
          <height>471</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item alignment="Qt::AlignHCenter">
            <widget class="QLabel" name="correlationQuestionLabel1">
             <property name="text">
              <string>Question 1</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="correlationQuestion1">
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <property name="isWrapping" stdset="0">
              <bool>false</bool>
             </property>
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item alignment="Qt::AlignHCenter">
            <widget class="QLabel" name="correlationAnswerLabel1">
             <property name="text">
              <string>Answer Choice 1</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="correlationAnswer1">
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::MultiSelection</enum>
             </property>
             <property name="isWrapping" stdset="0">
              <bool>false</bool>
             </property>
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>500</y>
          <width>381</width>
          <height>84</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QGroupBox" name="correlationTypeBox">
           <property name="title">
            <string>Correlation Type</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QRadioButton" name="positiveCorrelationButton">
              <property name="text">
               <string>Positive</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">correlationTypeGroup</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="negativeCorrelationButton">
              <property name="text">
               <string>Negative</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">correlationTypeGroup</string>
              </attribute>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="correlationSubmitButton">
           <property name="text">
            <string>Submit</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="showCorrelationsButton">
           <property name="text">
            <string>Show Correlations</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Probability Modifications</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget_2">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>10</y>
          <width>241</width>
          <height>471</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item alignment="Qt::AlignHCenter">
            <widget class="QLabel" name="probabilityQuestionLabel1">
             <property name="text">
              <string>Question</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="probabilityQuestion1">
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <property name="isWrapping" stdset="0">
              <bool>false</bool>
             </property>
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_11">
           <item alignment="Qt::AlignHCenter">
            <widget class="QLabel" name="probabilityAnswer1Label">
             <property name="text">
              <string>Answer Choice</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="probabilityAnswer1">
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::MultiSelection</enum>
             </property>
             <property name="isWrapping" stdset="0">
              <bool>false</bool>
             </property>
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget_6">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>500</y>
          <width>381</width>
          <height>84</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QGroupBox" name="correlationTypeBox2">
           <property name="title">
            <string>Probability Type</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_10">
            <item>
             <widget class="QRadioButton" name="positiveProbabilityButton">
              <property name="text">
               <string>Positive</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">probabilityTypeGroup</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="negativeProbabilityButton">
              <property name="text">
               <string>Negative</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">probabilityTypeGroup</string>
              </attribute>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="probabilitySubmitButton">
           <property name="text">
            <string>Submit</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="showProbabilitiesButton">
           <property name="text">
            <string>Show Probabilities</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="finishButton">
      <property name="text">
       <string>Finish</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="correlationTypeGroup"/>
  <buttongroup name="probabilityTypeGroup"/>
 </buttongroups>
</ui>
